.env.example

```
@@ -12,3 +12,17 @@ TEST_CLIENT_DB_PORT=5432
 TEST_CLIENT_DB_NAME=ecommerce_db
 TEST_CLIENT_DB_USER=monitoring_user
 TEST_CLIENT_DB_PASSWORD=1234
+
+
```

dotenv-linter (3.3.0)

[warning] [ExtraBlankLine] Extra blank line detected

(ExtraBlankLine)